portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,164 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+*{
+    box-sizing: border-box;
 }
 
-#greeting-container {
-  margin-top: 20px;
+nav {
+    background-color: black;
+    color: white;
+    text-align: center;
+    font-size: 24px;
+    width: 100%;
+    overflow: hidden;
+    position: sticky;
 }
+
+nav span {
+    float: left;
```

Comment:
I'm not a big fan of float. I recommend seeing if you can accomplish this using the flex rule. See https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
Ok cool I am going to take a look at the url you sent and see how to use flex to change all of my floats. I will note not to use float and use flex instead.